ng/src/web/pages/schedules/dialog.js

```diff
@@ -53,13 +53,18 @@ const DEFAULTS = {
 };
 
 const TimeUnitSelect = props => {
+  const unitOptions = [
+    {value: 'hour', label: _('hour(s)')},
+    {value: 'day', label: _('day(s)')},
+    {value: 'week', label: _('week(s)')},
+    {value: 'month', label: _('month(s)')},
+  ];
   return (
-    <Select {...props} defaultValue="hour">
-      <option value="hour">{_('hour(s)')}</option>
-      <option value="day">{_('day(s)')}</option>
-      <option value="week">{_('week(s)')}</option>
-      <option value="month">{_('month(s)')}</option>
-    </Select>
+    <Select
+      {...props}
+      items={unitOptions}
+      defaultValue="hour"
```

Comment:
Should be obsolete. Our Select component should not have a defaultValue prop.

Reply:
Deleted